pages.de/linux/pacman-files.md

```diff
@@ -0,0 +1,33 @@
+# pacman --files
+
+> Arch Linux Paket Management tool.
```

Comment:
Seems like a fine translation to me as well.

I based the translation of that line mainly on [the german apt page](https://github.com/tldr-pages/tldr/blob/main/pages.de/linux/apt.md?plain=1#L3) where its also called a "Paket Management Tool" _(though uppercase which I just noticed)_ though I think "Paketverwaltungs-Werkzeug" can also be a good translation, just wouldn't be consistent with the apt page.

Reply:
Translated pages are often inconsistent, because we can't really maintain a language we don't speak (well I do, but it's still a lot of work). Just go with what sounds best to you